### DISCUSSION TITLE
screensharing inside room

### DISCUSSION THREAD

**Discussion by katarzyna-z1** [mozilla/hubs#5994] [Spoke Questions]
Hello,
I use hubs blender exporter to create rooms, and I wonder if it's possible to add a link to screen-sharing inside blender .glb file, so when people enter the room the screen is already being shared?

**Comment by Exairnous** [comment]
I don't think that's currently possible, and I doubt it would be implemented because it could be a privacy concern.  There may be ways to alleviate the privacy concerns, but I think if you want this the best way would be to implement it in your own Hubs Cloud instance.  If you go this route, and can sort out the privacy concerns, a pull request would probably be welcome.

Disclaimer: I'm just a community member and this is just my own opinion.
